src/renderer/pages/repository/history.vue

```diff
@@ -83,15 +83,14 @@ export default {
 			return this.$store.getters["history/getFilePreview"];
 		}
 	},
-	mounted() {
-		this.gitLog();
+	async mounted() {
+		await this.gitLog();
```

Comment:
Instead of making an lifecycle hook async we can just use an async function in methods.